poet/__main__.py

```diff
@@ -1,6 +1,5 @@
-import sys
-
 from .poet import *
```

Comment:
This is in `__main__.py`, which is only loaded when the module is run like `python -m poet`. https://docs.python.org/2/using/cmdline.html#interface-options